pkg/pillar/devicenetwork/devicenetwork.go

```diff
@@ -254,6 +254,12 @@ func CheckDNSUpdate(ctx *DeviceNetworkContext) {
 		dnStatus = *ctx.DeviceNetworkStatus
 		status, _ := MakeDeviceNetworkStatus(*ctx.DevicePortConfig,
 			dnStatus)
+		// Copy port level cloud reachability flag
```

Comment:
Why is this not being done in MakeDeviceNetworkStatus itself? When the status is getting built? Why do we need a separate for loop to do this?